devtools/test_dashboard/server.js

```diff
@@ -3,54 +3,101 @@ var path = require('path');
 var http = require('http');
 var ecstatic = require('ecstatic');
 var open = require('open');
-var browserify = require('browserify');
+var webpack = require('webpack');
 var minimist = require('minimist');
 
 var constants = require('../../tasks/util/constants');
-var makeWatchifiedBundle = require('../../tasks/util/watchified_bundle');
-var shortcutPaths = require('../../tasks/util/shortcut_paths');
+var config = require('../../webpack.config.js');
+config.optimization = { minimize: false };
```

Comment:
Ideally even the test build would be minified, to give us further confidence we're testing the behavior of the real bundle, plus a sourcemap. Right now with no sourcemap all I see in the test dashboard is the single huge unminified bundle - all the code is there unmodified but it'd be a pain to connect it back to the source files during debugging.

Reply:
The debugging is addressed in [458921f](https://github.com/plotly/plotly.js/pull/6355/commits/458921f44f95590e4775523f03d07f8c67d10661).
Now using *development* `mode`, the bundle content is different from *production* mode; but it builds & rebuilds faster.
This should be good for now. And we could further improve this configuration.

Reply:
That'll work - what you see in the debugger isn't quite the same as the source files - it's been reformatted a bit and the require statements are transformed - but it's pretty close and easy to move from one to another 😄